local-dev/templates/docker-rsync-local-orch.sh

```diff
@@ -107,7 +107,7 @@ start_server () {
     -v sbt-cache:/root/.sbt \
     -v jar-cache:/root/.ivy2 \
     -v coursier-cache:/root/.cache/coursier \
-    --add-host local.broadinstitute.org:$DOCKERHOST \
+    --add-host local.dsde-dev.broadinstitute.org:$DOCKERHOST \
     -p 5051:5051 \
     --network=fc-orch \
     -e JAVA_OPTS="$DOCKER_JAVA_OPTS" \
```

Comment:
the line just below this (which I can't comment on via the github UI) is incorrect. See https://github.com/broadinstitute/firecloud-orchestration/pull/1444 … the docker image tag needs an update to `eclipse-temurin-jammy-17.0.10_7_1.10.2_2.13.15`.

Maybe this PR just needs to be rebased against `develop`